Modix.Bot/Modules/EmojiStatsModule.cs

```diff
@@ -130,7 +129,7 @@ await ReplyAsync(embed: new EmbedBuilder()
                 percentUsage = 0;
 
             var numberOfDays = Math.Clamp((DateTime.Now - guildStats.OldestTimestamp).Days, 1, 30);
-            var perDay = (double)emojiStats30.Uses / numberOfDays;
+            var perDay = (double)emojiStats.Uses / numberOfDays;
```

Comment:
Is this not going to generate garbage data? By my reckoning, `emojiStats.Uses` is now an all-time use-count, while `numberOfDays` is still clamped to a max of 30.